.github/workflows/ci.yaml

```diff
@@ -29,33 +29,12 @@ jobs:
       with:
         version: ${{ matrix.node_version }}
     - name: Install dependencies
-      run: |
```

Comment:
@musculman Looks like we don't need custom install dependencies step at all now.

Reply:
Happy to remove code!